ui/utils.py

```diff
@@ -62,12 +67,11 @@ def user_moira_lists(user):
         list: A list of moira lists the user has access to.
     """
     moira_user = get_moira_user(user)
+    moira = get_moira_client()
     try:
-        moira = get_moira_client()
         return moira.user_lists(moira_user.username, moira_user.type)
     except Exception as exc:  # pylint: disable=broad-except
-        log.exception('Something went wrong with the moira client for user %s: %s', user.email, str(exc))
-        return []
+        raise MoiraException('Something went wrong with getting moira lists for user %s: %s', user.email, str(exc))
```

Comment:
See exception chaining comment above. To put the user email in the exception message you'll need to use `.format()` or use a format string

Reply:
OK